express/tests/validators/emailValidator.test.ts

```diff
@@ -0,0 +1,37 @@
+import {expect} from "chai";
+import {validateEmail} from "../../src/lib/emailValidator";
+
+const EMAIL_WITH_ALLOWED_DOMAIN = "allowed@email.gov.uk";
+const EMAIL_WITH_DISALLOWED_DOMAIN = "allowed@hackers.co.uk";
+
+const EMAIL_WITH_NO_AT_SYMBOL = "copyAndPaste_at_some.domain.gov.uk";
+const EMAIL_COMPLYING_WITH_RFC822 = "an_Email+modification@some.domain.gov.uk";
```

Comment:
There was one other test with an underscore after the @ symbol. Can we restore it?